controller/tap/server.go

```diff
@@ -334,7 +334,10 @@ func NewServer(addr string, tapPort uint, kubeconfig string) (*grpc.Server, net.
 			return nil, fmt.Errorf("Object is not a Pod")
 		}
 		deployment, err := replicaSets.GetDeploymentForPod(pod)
-		return []string{deployment}, err
+		if err != nil {
+			return []string{}, nil
+		}
+		return []string{deployment}, nil
```

Comment:
This change appears to result in no output from the tap subcommand. Need to update the ReplicaSet store to fix.

Reply:
This appears to be an issue in master as well. I opened #136 to track it.